### ISSUE TITLE
“厌恶”功能

### ISSUE THREAD

**Issue by Ahrwing** [Ahrwing/mcmod#487]
既然可以设定“喜欢”的MOD，何不追加一个“厌恶”的MOD？

- 被厌恶的MOD不会在当前登录用户的视野中出现，但可以通过地址访问。
- 可以在厌恶列表中取消厌恶。

**Comment by limuness** [comment]
Good idea,I like it.and then can add a blacklist function(user)?Will be  prohibit him in my homepage message and exchange.

**Comment by rsp4jack** [comment]
为什么会有一个人对一个 Mod 恨之入骨？

**Comment by HanlieChina** [comment]
额外
主页随便看看：忽略模组/忽略标签/忽略模组元素
感觉可以作为主页v4的更新点。
